gui/app/stores/MemStore.scala

```diff
@@ -17,4 +17,10 @@ class MemStore(val edges: List[Edge], val nodes: List[Node]) extends Store {
 
   final override def getMatchingNodesCount(text: String): Int =
     getMatchingNodes(limit = Integer.MAX_VALUE, offset = 0, text = text).size
+
+  final override def getTotalNodesCount(): Int =
```

Comment:
Scala convention is that nullary methods that have side effects are declared with () and those without side effects are not. So 
```
def getTotalNodesCount: Int
```

Reply:
I fixed this.

Reply:
Oh interesting, will keep in mind